codec-http/src/main/java/io/netty/handler/codec/http/HttpServerUpgradeHandler.java

```diff
@@ -61,9 +61,10 @@
 
         /**
          * Adds any headers to the 101 Switching protocols response that are appropriate for this protocol.
+         * Returns {@code true} if an upgrade was done, {@code false} otherwise.
```

Comment:
How about:

```
Performs the preparatory steps required for performing a protocol update. This method returns a boolean value
to proceed or abort the upgrade in progress. If {@code false} is returned, the upgrade is aborted and
the {@code upgradeRequest} will be passed through the inbound pipeline as if no upgrade was performed.
If {@code true} is returned, the upgrade will proceed to the next step which invokes {@link #upgradeTo}.
When returning {@code true}, you can add headers to the {@code upgradeHeaders} so that they are added
to the 101 Switching protocols response.
```

I'm not a native speaker, so please feel free to fix this as well.